src/assets/cities.json

```diff
@@ -0,0 +1,64 @@
+{
+    "cities": [
```

Comment:
Change naming, it isn't "cities" listing. Also, ask PO if you don't need to change in job registration form field label "City" to "Municipality" or "Region".